src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -59,6 +60,7 @@ def __init__(self, appname: str = "zowe", show_warnings: bool = True):
         self.project_user_config = ConfigFile(type=USER_CONFIG, name=appname)
 
         self.__logger = logging.getLogger(__name__)
+        Log.registerLogger(__name__)
```

Comment:
Thanks for adding the `registerLogger` function 👍  Sorry I wasn't very clear in my previous comment, but would it make sense to have the `registerLogger` function return the same logger returned by `getLogger`? That would allow us to condense these 2 lines into just 1:
```suggestion
        self.__logger = Log.registerLogger(__name__)
```